Telecominfraproject/OpenCellular
# Environment Detection

## Summary
firmware/utilities/build includes a shell script that detects the build environment. It works in both Windows and Linux.

## Test Plan
1. Build from firmware/ec. Verify that `firmware/utilities/build/build_ec_cfg.txt` is created and firmware builds with no new issues. The text file should be created even if the build fails.
2. Build from firmware/host. Verify that `firmware/utilities/build/build_host_cfg.txt` is created and host builds with no new issues. The text file should be created even if the build fails.
3. Build from firmware/ec/test.  Verify that `firmware/utilities/build/build_ut_cfg.txt` is created and firmware builds with no new issues. The text file should be created even if the build fails.

## Issues
Resolved issue #171
